packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -405,6 +411,7 @@ export abstract class SharedObjectCore<
 	 * and not sent to the server. This will be sent back when this message is received back from the server. This is
 	 * also sent if we are asked to resubmit the message.
 	 */
+	// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- TODO: use unknown instead of any (legacy breaking)
```

Comment:
Should we file a work item to fix these changes that would be legacy breaking for the next legacy breaking release?

Reply:
https://github.com/microsoft/FluidFramework/issues/23236